.golangci.yml

```diff
@@ -9,23 +9,105 @@ linters-settings:
   gofmt:
     # simplify code: gofmt with `-s` option, true by default
     simplify: true
+  tagliatelle:
+    case:
+      rules:
+        json: snake
+  whitespace:
+    multi-func: true
+    multi-if: true
+  gosec:
+    excludes:
+      - G402 # Look for bad TLS connection settings.
+      - G306 # Poor file permissions used when writing to a new file.
+  staticcheck:
+    go: "1.17"
```

Comment:
Q: does static check not support go "1.18" yet? Would this fail if we add any generic function because the tool will fail to parse it?